module/behaviour/Controller/src/Controller.cpp

```diff
@@ -92,7 +92,9 @@ namespace module::behaviour {
             [this](const ActionPriorities& update) {
                 auto& request = requests[update.id];
 
-                // Find the largest priority
+                // TODO(BehaviourTeam): This can be converted to min_element
+                // ! in order to have the highest priority being
+                // ! 0 rather than the upper limit being extremely high Find the largest priority
```

Comment:
My brain immediately negates anything behind an exclamation point in this sort of context. Are these there for a reason?

Reply:
![Screenshot from 2022-07-08 12-38-01](https://user-images.githubusercontent.com/63719859/177906883-14819259-752d-43ae-b83e-e6a187d73c34.png)